src/render/src/macros/mod.rs

```diff
@@ -106,20 +106,22 @@ macro_rules! gfx_format {
 /// The `pipeline state object` or `pso` can consist of the following
 /// `pso` components:
 ///
-/// - A single [vertex buffer](pso/buffer/type.VertexBuffer.html) component to hold the vertices.
-/// - A single [instance buffer](pso/buffer/type.InstanceBuffer.html) component.
+/// - A [vertex buffer](pso/buffer/type.VertexBuffer.html) component to hold the vertices.
+/// - An [instance buffer](pso/buffer/type.InstanceBuffer.html) component.
 /// - Single or multiple [constant buffer](pso/buffer/struct.ConstantBuffer.html) components. (DX11 and OpenGL3)
-/// - Single or multiple [global buffer](pos/buffer/struct.Global.html) components.
-/// - Single or multiple [samplers](pos/resource/struct.Sampler.html) 
-///   such as [texture samplers](pso/resource/struct.TextureSampler.html].
+/// - Single or multiple [global buffer](pso/buffer/struct.Global.html) components.
+/// - Single or multiple [samplers](pso/resource/struct.Sampler.html).
 /// - [Render](pso/target/struct.RenderTarget.html), [blend](pso/target/struct.BlendTarget.html), 
-///   [depth](pso/target/struct.DepthTarget.html) and [stencil](pso/target/struct.StencilTarget.html) targets
+///   [depth](pso/target/struct.DepthTarget.html), [stencil](pso/target/struct.StencilTarget.html) targets.
 /// - A [shader resource view](pso/resource/struct.ShaderResource.html) (SRV, DX11)
-/// - A [unordered access view](pso/resource/struct.UnorderedAccess.html) (UAV, DX11, not yet implemented in the OpenGL backend)
+/// - An [unordered access view](pso/resource/struct.UnorderedAccess.html) (UAV, DX11, not yet implemented in the OpenGL backend)
 /// - A [scissor](pso/target/struct.Scissor.html) rectangle value (DX11)
 ///
-/// Structure of a `pipeline state object` can be defined
-/// freely but should at the very least consist of one vertex buffer object.
+/// Structure of a `pipeline state object` can be defined freely.
+///
+/// It should be noted however, that you can have multiple objects of everything but
```

Comment:
I think we should call them "components" instead of "objects" here for consistency.